lib/utils/logger.js

```diff
@@ -67,12 +94,11 @@ Object.freeze(methods);
  * Logger function.
  * @param {*} method
  */
-const log = method => (...args) => {
+const log = (method) => (...args) => {
```

Comment:
Lets describe two args here (text, deviceId) for clarity, and pass text on line 122 instead of ...args, so first argument is always message text and second deviceId?